core/models/user.py

```diff
@@ -106,6 +108,7 @@ class UserFeedback(models.Model):
                                    choices=option,
                                    null=False)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    resolved = models.BooleanField(default=False)
 
     class Meta:
         verbose_name_plural = "User Feedback"
```

Comment:
Add tests for the `feature request` report type.

Reply:
@collinmutembei i don't understand what we are supposed to test for here

Reply:
@collinmutembei done, but coverage has decreased & i'm unable to locate the file causing the reduction

Reply:
@collinmutembei done